disnake/guild.py

```diff
@@ -4313,8 +4313,10 @@ async def create_automod_rule(
             The type of trigger that determines whether this rule's actions should run for a specific event.
             If set to :attr:`~AutoModTriggerType.keyword` or :attr:`~AutoModTriggerType.keyword_preset`,
             ``trigger_metadata`` must be set accordingly.
-        actions: Sequence[:class:`AutoModAction`]
+            This cannot be changed after creation.
+        actions: Sequence[Union[:class:`AutoModBlockMessageAction`, :class:`AutoModSendAlertAction`, :class:`AutoModTimeoutAction`, :class:`AutoModAction`]]
```

Comment:
This list of options would make sense to be defined like `|resource_type|` was defined, IMO.

Reply:
👍